features/transformstream-transformer-cancel.yml

```
@@ -0,0 +1,6 @@
+name: "TransformStream: transformer.cancel method"
```

To align with our naming conventions here:

```suggestion
name: TransformStream transformer cancel() method
```